README.md

```diff
@@ -43,3 +43,17 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
```

Comment:
- [ ] Please explain in more detail how a user can clone and start your project, include how it should create and migrate the databases